lib/cc/cli/analyze.rb

```diff
@@ -72,6 +64,27 @@ def process_args
       def formatter
         @formatter ||= Formatters::PlainTextFormatter.new(filesystem)
       end
+
+      def disable_all_engines!
+        unless engines_disabled
+          config.engines.each { |e| e.enabled = false }
+          @engines_disabled = true
+        end
+      end
+
+      def enable_engine(name, channel)
+        existing_engine = config.engines.detect { |e| e.name == name }
+        if !existing_engine.present?
```

Comment:
What do you think of flipping this so that `if existing_engine.present?` is the first block?

Reply:
Done